Help glad find local headers:
3rdparty/glad/include/glad_egl.h:212:10: fatal error: 'KHR/khrplatform.h' file not found

--- 3rdparty/glad/CMakeLists.txt.orig	2022-09-02 10:21:30 UTC
+++ 3rdparty/glad/CMakeLists.txt
@@ -12,6 +12,7 @@ elseif(X11_API OR WAYLAND_API)
 	)
 endif()
 
+target_include_directories(glad PRIVATE "${CMAKE_INSTALL_PREFIX}/include")
 target_include_directories(glad PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/include")
 target_include_directories(glad INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/include")
 target_link_libraries(glad PRIVATE Threads::Threads "${CMAKE_DL_LIBS}")
